sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -263,6 +263,23 @@ class wi_element<sycl::ext::oneapi::bfloat16, NumRows, NumCols, Use, Layout,
 #endif // __SYCL_DEVICE_ONLY__
   }
 
+  operator float() {
```

Comment:
This already exists in https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/bfloat16.hpp#L49